src/datatrove/utils/word_tokenizers.py

```diff
@@ -74,9 +85,201 @@ def span_tokenize(self, text: str) -> list[tuple[int, int]]:
+    Languages.marathi: lambda: SpaCyTokenizer("mr"),
+    Languages.telugu: lambda: SpaCyTokenizer("te"),
+    Languages.hungarian: lambda: SpaCyTokenizer("hu"),
+    Languages.romanian: lambda: SpaCyTokenizer("ro"),
+    Languages.ukrainian: lambda: SpaCyTokenizer("uk"),
+    Languages.slovak: lambda: SpaCyTokenizer("sk"),
+    Languages.bulgarian: lambda: SpaCyTokenizer("bg"),
+    Languages.catalan: lambda: SpaCyTokenizer("ca"),
+    Languages.croatian: lambda: SpaCyTokenizer("hr"),
+    Languages.latin: lambda: SpaCyTokenizer("la"),
+    Languages.serbian: lambda: SpaCyTokenizer("sr"),
+    Languages.lithuanian: lambda: SpaCyTokenizer("lt"),
+    Languages.hebrew: lambda: SpaCyTokenizer("he"),
+    Languages.latvian: lambda: SpaCyTokenizer("lv"),
+    Languages.icelandic: lambda: SpaCyTokenizer("is"),
+    Languages.armenian: lambda: SpaCyTokenizer("hy"),
+    Languages.basque: lambda: SpaCyTokenizer("eu"),
+    Languages.thai: lambda: ThaiTokenizer(),
+    Languages.georgian: lambda: GeorgianTokenizer(),
+    Languages.tagalog: lambda: SpaCyTokenizer("tl"),
+    Languages.albanian: lambda: SpaCyTokenizer("sq"),
+    Languages.macedonian: lambda: SpaCyTokenizer("mk"),
+    Languages.azerbaijani: lambda: SpaCyTokenizer("az"),
+    Languages.amharic: lambda: SpaCyTokenizer("am"),
+    Languages.bengali: lambda: SpaCyTokenizer("bn"),
+    Languages.malay: lambda: SpaCyTokenizer("ms"),
+    Languages.urdu: lambda: SpaCyTokenizer("ur"),
+    Languages.nepali: lambda: SpaCyTokenizer("ne"),
+    Languages.kazakh: lambda: StanzaTokenizer("kk"),
+    Languages.gujarati: lambda: SpaCyTokenizer("gu"),
+    Languages.kannada: lambda: SpaCyTokenizer("kn"),
+    Languages.welsh: lambda: StanzaTokenizer("cy"),
+    Languages.norwegian_nynorsk: lambda: NLTKTokenizer("norwegian"),
+    Languages.sinhala: lambda: SpaCyTokenizer("si"),
+    Languages.tatar: lambda: SpaCyTokenizer("tt"),
+    Languages.afrikaans: lambda: SpaCyTokenizer("af"),
+    Languages.kirghiz: lambda: SpaCyTokenizer("ky"),
+    Languages.irish: lambda: SpaCyTokenizer("ga"),
+    Languages.luxembourgish: lambda: SpaCyTokenizer("lb"),
+    Languages.maltese: lambda: StanzaTokenizer("mt"),
+    Languages.sanskrit: lambda: SpaCyTokenizer("sa"),
+    Languages.yoruba: lambda: SpaCyTokenizer("yo"),
+    Languages.serbocroatian: lambda: SpaCyTokenizer("sr"),
+    Languages.oriya: lambda: IndicNLPTokenizer("or"),
+    Languages.punjabi: lambda: IndicNLPTokenizer("sa"),
+    Languages.assamese: lambda: IndicNLPTokenizer("as"),
+    Languages.war: lambda: IndicNLPTokenizer("war"),
+    Languages.sindhi: lambda: IndicNLPTokenizer("sd"),
+    Languages.bosnian: lambda: SpaCyTokenizer("hr"),  # Proxy
+    Languages.belarusian: lambda: SpaCyTokenizer("uk"),  # Proxy
+    Languages.galician: lambda: NLTKTokenizer("portuguese"),  # Proxy
+    Languages.esperanto: lambda: NLTKTokenizer("english"),  # Proxy
+    Languages.occitan: lambda: NLTKTokenizer("italian"),  # Proxy
```

Comment:
```suggestion
    Languages.occitan: lambda: SpaCyTokenizer("ca"),  # Proxy
```
use catalan instead as its the closest language